docs/PORTS.md

```diff
@@ -0,0 +1,118 @@
+<sup>3</sup>Refers only to the Focus Mk4 (C519) available in Europe/China/Taiwan/Australasia, not the Focus Mk3 (C346) in North and South America/Southeast Asia. <br />
+<sup>4</sup>Requires a <a href="https://github.com/commaai/openpilot/wiki/GM#hardware" target="_blank">community built ASCM harness</a>. <b><i>NOTE: disconnecting the ASCM disables Automatic Emergency Braking (AEB).</i></b> <br />
+<sup>5</sup>2019 Honda Civic 1.6L Diesel Sedan does not have ALC below 12mph. <br />
+<sup>6</sup>Requires a comma 3X or <a href="https://comma.ai/shop/can-fd-panda-kit" target="_blank">CAN FD panda kit</a> for this <a href="https://en.wikipedia.org/wiki/CAN_FD" target="_blank">CAN FD car</a>. <br />
+<sup>7</sup>In the non-US market, openpilot requires the car to come equipped with EyeSight with Lane Keep Assistance. <br />
+<sup>8</sup>openpilot operates above 28mph for Camry 4CYL L, 4CYL LE and 4CYL SE which don't have Full-Speed Range Dynamic Radar Cruise Control. <br />
+<sup>9</sup>Not including the China market Kamiq, which is based on the (currently) unsupported PQ34 platform. <br />
+<sup>10</sup>Refers only to the MQB-based European B8 Passat, not the NMS Passat in the USA/China/Mideast markets. <br />
+<sup>11</sup>Some Škoda vehicles are equipped with heated windshields, which are known to block GPS signal needed for some comma three functionality. <br />
+<sup>12</sup>Only available for vehicles using a gateway (J533) harness. At this time, vehicles using a camera harness are limited to using stock ACC. <br />
+<sup>13</sup>Model-years 2022 and beyond may have a combined CAN gateway and BCM, which is supported by openpilot in software, but doesn't yet have a harness available from the comma store. <br />
+
+## Community Maintained Cars
+Although they're not upstream, the community has openpilot running on other makes and models. See the 'Community Supported Models' section of each make [on our wiki](https://wiki.comma.ai/).
+
+* Honda
+  * Honda Clarity
+
+    The Clarity has one extra CAN bus at the camera connector. Due to this, it needs custom hardware and a custom fork to run.
+
+    Join the #honda-acura channel on [Discord](https://discord.comma.ai) with questions.
+  * Serial Steering
+
+    There are some Honda/Acura with Honda Sensing/AcuraWatch Plus that are not currently supported due to using dedicated serial data lines for its steering control messages. These cars can work with openpilot using additional hardware and minor software modifications. A Community Maintained upstream is planned.
+
+    The current list of 'serial steering' cars:
+
+    * 2016/2017 Accord
+    * 15-20 Acura MDX (2022+ MDX does not use serial steering and uses the same Bosch harness, but the wiring pinout is completely different)
+    * 15-20 Acura TLX (2021+ TLX uses the same Bosch harness with the same wiring pinout)
+
+    Join the #topic-serial-steering channel on discord with questions.
+
+
+
+# Don't see your car here?
```

Comment:
FYI, think we link to this on web3

Reply:
We already do link to this section, on CARS.md.